.github/dependabot.yml

```diff
@@ -1,8 +1,17 @@
 version: 2
 updates:
-    - package-ecosystem: 'npm'
-      directory: '/'
+    - package-ecosystem: npm
+      directory: /
       schedule:
-          interval: 'weekly'
+          interval: weekly
       open-pull-requests-limit: 5
-      versioning-strategy: 'increase'
+      versioning-strategy: increase
+      groups:
+          dev-dependencies:
+              applies-to: version-updates
+              dependency-type: development
+              update-types:
+                  - minor
+                  - patch
```

Comment:
I agree that excluding `major` here is a good choice. When grouping things together it becomes harder to track down regressions to an individual lib so excluding version bumps with breaking changes makes sense.